examples/nodes/class_def.py

```diff
@@ -13,7 +13,7 @@ class definition.
         - The contents of the class body.
     - decorators            (Optional[Decorators])
         - The decorator to be applied on this function.
-    - doc                   (Optional[str])
+    - doc_node                   (Optional[Const])
```

Comment:
Please delete some spaces to align the `(Optional` with the other ones. Same for the `function_def.py` file